content/zh-cn/blog/_posts/2017-10-00-Kubernetes-Community-Steering-Committee-Election-Results.md

```diff
@@ -2,16 +2,14 @@
 title: " Kubernetes 社区指导委员会选举结果 "
 date: 2017-10-05
 slug: kubernetes-community-steering-committee-election-results
-url: /zh/blog/2017/10/Kubernetes-Community-Steering-Committee-Election-Results
```

Comment:
Is it wrong?

Reply:
yes
Chinese url: https://kubernetes.io/zh/blog/2017/10/Kubernetes-Community-Steering-Committee-Election-Results/
English URL: https://kubernetes.io/blog/2017/10/kubernetes-community-steering-committee-election-results/